### ISSUE TITLE
classify new sample

### ISSUE THREAD

**Issue by chenyafe052** [hfawaz/dl-4-tsc#46]
What is the purpose of the 'predict' function in the classifiers files? it's look that this function not in use at all.

In addition, I would appreciate your help with making classification for a new sample with the pre-trained model.

**Comment by Gpialla** [comment]
The *predict* function is used to evaluate a model on the test set.
It also saves the results in a DataFrame in order to save them late.

To classify new samples you can do something like:
```
model = Classifier_INCEPTION.build_model(input_shape, nb_classes)
model.load_weights('path/to/the/weights')
model.predict(x)
```
